### PR TITLE
Snakefile for SRA downloads

### PR DESCRIPTION
**Pull request by lkhilton**
I created this Snakefile to download fastq or sam files from SRA/dbGaP and handle compression to .gz/.bam/.cram.

### CODE REVIEW

File: sra-tools/1.0/sra_dump.smk [hunk 1]
```diff
@@ -0,0 +1,149 @@
+import pandas as pd
+import os
+
+configfile: "test_sra_dump.yaml"
```
Comment (Kdreval) [inline sra-tools/1.0/sra_dump.smk:4]: Should this be renamed to the `sra_dump.yaml` (without `test`)?

File: sra-tools/1.0/sra_dump.smk [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+    resources:
+        **config["resources"]["sam_to_bam"]
+    shell:
+        "samtools view {params.bam_cram} -@ {threads} {input.sam} > {output.bam}"
+
+rule index_bam:
+    input:
+        bam = str(rules.sam_to_bam.output.bam)
+    output:
+        complete = touch("{accession}.indexed")
```
Comment (Kdreval) [inline sra-tools/1.0/sra_dump.smk:127]: I just wonder why this is not a designated index extension, because we don't know upfront if this is bam or cram?

**Review by Kdreval — COMMENTED** [review]
Thanks!

File: sra-tools/1.0/sra_dump.smk (hunk 1 above)
Comment (lkhilton) [reply sra-tools/1.0/sra_dump.smk]: Good catch - fixed.

**Review by Kdreval — APPROVED** [review]
Thanks!

**Pull request merged**